packages/contracts-bedrock/test/OptimismMintableERC20Factory.t.sol

```diff
@@ -57,7 +57,7 @@ contract OptimismMintableTokenFactory_Test is Bridge_Initializer {
         view
         returns (address)
     {
-        bytes memory constructorArgs = abi.encode(address(L2Bridge), _remote, _name, _symbol);
```

Comment:
We should have decimals be an arg to `calculateTokenAddress` instead of hardcoding it so that its portable to be used